.gitignore

```diff
@@ -11,4 +11,5 @@ build/
 
 .classpath
 .project
-.settings/
\ No newline at end of file
+.settings/
+/target/
```

Comment:
Is this file required ??

Reply:
yes we are ignoring these folder while pushing to git from local